.travis.yml

```diff
@@ -69,6 +69,8 @@ jobs:
     - python -m pip install --user --upgrade tox virtualenv==20.0.23
     # Install quart to run the http2 tests.
     - python -m pip install --user quart
+    # DISPATCH-1883: Install selectors to run tcp echo server/client tools
+    - python -m pip install --user selectors
```

Comment:
is this something that the GitHub Action also needs?

Reply:
yes. well spotted.